types/react-native/test/index.tsx

```diff
@@ -877,8 +878,24 @@ const deviceEventEmitterStatic: DeviceEventEmitterStatic = DeviceEventEmitter;
 deviceEventEmitterStatic.addListener('keyboardWillShow', data => true);
 deviceEventEmitterStatic.addListener('keyboardWillShow', data => true, {});
 
-const nativeEventEmitter = new NativeEventEmitter();
-nativeEventEmitter.removeAllListeners('event');
+// NativeEventEmitter - Android
+const androidEventEmitter = new NativeEventEmitter();
+let sub1 = androidEventEmitter.addListener('event', (event: object) => event);
```

Comment:
use const instead of let or the CI will yell at you because it's not getting reassigned. (eslint: prefer-const). same for othe usage of let.

Reply:
Updated let/const.